app/services/record_declarations/base.rb

```diff
@@ -140,11 +140,23 @@ def validate_schedule_present
     end
 
     def milestone
-      schedule&.milestones&.find_by(declaration_type: declaration_type)
+      milestone_schedule && milestone_schedule.milestone
     end
```

Comment:
What is this returning now? Is milestone being called anywhere else in this file that this update might impact?

Reply:
this will return either `nil` with the `milestone`. Just like before 💅

Reply:
👍